mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/core/zookeeper/CuratorService.java

```diff
@@ -64,10 +61,6 @@ public CuratorService(CoreConfiguration configs) {
                 .retryPolicy(new ExponentialBackoffRetry(configs.getZkConnectionRetrySleepMs(), configs.getZkConnectionMaxRetries()))
                 .connectString(configs.getZkConnectionString())
                 .build();
-
-        masterMonitor = new ZookeeperMasterMonitor(
```

Comment:
For me this was a convenience method that was no longer convenient. In my mental model the ZK master monitor needs a curator framework, but the curator doesn't necessarily need to know about the master monitor.